public/app/features/search/page/components/RootFolderView.tsx

```diff
@@ -33,33 +49,22 @@ export const FolderView = ({
   const styles = useStyles2(getStyles);
 
   const results = useAsync(async () => {
-    const folders: DashboardViewItem[] = [];
+    const folders = await getChildren();
 
     if (!hidePseudoFolders) {
-      if (contextSrv.isSignedIn) {
-        const stars = await getBackendSrv().get('api/user/stars');
-        if (stars.length > 0) {
-          folders.push({ title: 'Starred', icon: 'star', kind: 'folder', uid: '__starred', itemsUIDs: stars });
-        }
-      }
+      folders.unshift({ title: 'General', url: '/dashboards', kind: 'folder', uid: GENERAL_FOLDER_UID });
```

Comment:
this is now inside the `!hidePseudoFolders` condition, whereas previously it was outside

Reply:
Oh, this probably explains it. I've been looking at this, my diff, and all the behaviour and really failing to understand why the tests are failing 😅

Reply:
Fixed general folder not showing 👍🏻